### ISSUE TITLE
Error when creating CRUD for model company

### ISSUE THREAD

**Issue by brianmcaudill** [TiagoSilvaPereira/vemto-issues#321]
![_exported_image](https://user-images.githubusercontent.com/329199/169080127-d5e6be4e-c3db-49d1-b984-44747b46b60b.png)
```
TypeError: Cannot read property 'type' of undefined
    at Function.getCorrectInputFromFieldAndCrud (app://./js/chunk-common.d5e10964.js:1:64716)
    at Proxy.addInputToCrud (app://./js/chunk-common.d5e10964.js:1:24879)
    at app://./js/chunk-common.d5e10964.js:1:33662
    at Array.forEach (<anonymous>)
    at Proxy.addInputsFromModel (app://./js/chunk-common.d5e10964.js:1:33649)
    at Function.fromModel (app://./js/chunk-common.d5e10964.js:1:32986)
    at o.create (app://./js/project_apps.c4309195.js:1:12235)
    at app://./js/project_apps.c4309195.js:1:12207
```

---

vemto 1.1.4
Electron 8.5.5
win32 10.0.18363
Locale: en-US

**Comment by TiagoSilvaPereira** [comment]
Hi @brianmcaudill there is a problem with the *foreignId* type... please check #283 for a workaround... I"ll fix it soon

**Comment by TiagoSilvaPereira** [comment]
Fixed on 1.1.5
